tests/launchChecklist.spec.js

```
@@ -0,0 +1,219 @@
+      {
+        area: 'Security',
+        metric: 'Dependency Vulnerabilities',
+        current: '2 moderate (dev-only)',
+        target: '0',
+        recommendations: ['Update Vite to latest version', 'Update esbuild dependency']
+      }
+    ];
+  }
+};
+
+module.exports = { launchChecklist };
```

Biome (2.1.2)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)